universal-login-relayer/lib/utils/utils.js

```diff
@@ -42,6 +44,18 @@ const withENS = (provider, ensAddress) => {
   return new providers.Web3Provider(provider._web3Provider, chainOptions);
 };
 
+const isEnoughGasLimit = (estimateGas, gasLimit) =>
+  gasLimit >= estimateGas;
```

Comment:
No need for a separate method can be inlined.

Reply:
✅